### DISCUSSION TITLE
數字比大小 -- submit error

### DISCUSSION THREAD

**Discussion by maxchen1220** [Lidemy/forum#148] [General]
想請問一下爲什麼怎麼試都是error.我的code哪部分有問題呢？謝謝

import sys
lines = sys.stdin.readline().strip('\n')

def solve(tmp):
	tmp = str.split()
	a = int(tmp[0])
	b = int(tmp[1])
	if (a == 0) & (b == 0):
		return
	if a > b:
		print(a)
	else:
		print(b)

**Comment by aszx87410** [comment]
因為輸入有很多行，不是只有一行
